ENVIRONMENT.md

```diff
@@ -5,6 +5,28 @@ the maintainers if you do not have them and feel you should be given access. The
 way to define them for use in R is in an `.Renviron` file, either global or local.
 You can directly do this in R with `usethis::edit_r_environ()`.
 
+## Generating signals
+
+Environment variables are used to determine how `generate_signals()` works. These
+are all logical variables that are read internally to determine how the underlying
+functions behave and the final output returned. The values should be `TRUE` or `FALSE`.
+
+- `HS_LOCAL`: if `TRUE`, prevents functions from writing to the Azure containers,
+sending emails with Mailchimp, or calling the OpenAI endpoint. All other functionality
+occurs, but the outputs thus remain local to the running machine. Useful for testing
+and interactive development.
+- `HS_DRY_RUN`: if `TRUE`, `generate_signals()` does a dry run, which ensures
```

Comment:
Would clarify that in the dry run, you're guaranteed to have signals generated for testing.

Reply:
https://github.com/OCHA-DAP/hdx-signals/pull/211/commits/95cff7c72893d8052891ae23a3e630c2f39253f0